File: __archive/leet/minDeletionSize.cpp
// problem 945
#include "vector"
using namespace std;

class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        int countt = 0;
        for(int j = 0; j < strs[0].length(); j++){
            for(int i = 0; i < strs.size() - 1; i++){
                if(strs[i + 1][j] < strs[i][j]){
                    countt++;
                    break;
                }  
            }
        }
        return countt;      
    }
};

File: __archive/leet/minDeletionSize.py
# problem 945
from typing import List

class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        countt = 0
        for j in range(len(strs[0])):
            for i in range(len(strs) - 1):
                if strs[i + 1][j] < strs[i][j]:
                    countt += 1
                    break
        
        return countt 